src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -537,3 +548,29 @@ def publish_cluster_nodes(self, node_count: int):
             self.handle_supabase_error()
 
         logging.info(f"cluster nodes: {UPDATE_CLUSTER_NODE_COUNT} => {data}")
+
+    def persist_events_block(self, block: EventsBlock, finding_id):
+        db_events = []
+        for row in block.render_rows():
+            event = dict(zip(block.headers, row))
+            event["account_id"] = self.account_id
+            event["cluster_id"] = self.cluster
+            event["namespace"] = block.namespace
+            if finding_id:
+                event["finding_id"] = str(finding_id)
+            event.setdefault("kind", block.kind)
+            event.setdefault("name", block.resource_name)
+            db_events.append(event)
+
+        res = self.client.table(RESOURCE_EVENTS).insert(db_events, upsert=True).execute()
```

Comment:
will upserting override the previous events row stored in the table?
like if we get the events for deployment A today and again tomorow will there only be the newer events?